apps/avifenc.c

```diff
@@ -805,6 +792,63 @@ int main(int argc, char * argv[])
         ++argIndex;
     }
 
+    // Check lossy/lossless parameters and set to default if needed.
+    if (lossless) {
+        // Pixel format.
+        if (input.requestedFormat != AVIF_PIXEL_FORMAT_NONE && input.requestedFormat != AVIF_PIXEL_FORMAT_YUV444) {
+            fprintf(stderr,
+                    "When set, the pixel format can only be 444 in lossless "
+                    "mode.\n");
+            returnCode = 1;
+        }
+        input.requestedFormat = AVIF_PIXEL_FORMAT_YUV444; // don't subsample when using
+                                                          // AVIF_MATRIX_COEFFICIENTS_IDENTITY
```

Comment:
Nit: I suggest moving the comment before the assignment statement.

```
        // don't subsample when using AVIF_MATRIX_COEFFICIENTS_IDENTITY
        input.requestedFormat = AVIF_PIXEL_FORMAT_YUV444;
```

In the original code, there are multiple assignment statements and the comments are placed after the statements. We don't need to do that in the new code.

Reply:
Done